test/format/format_disagg_multi.sh

```diff
@@ -40,7 +40,7 @@ fi
 
 PROG="./t"
 CONFIG="../../../test/format/CONFIG.disagg"
-DISAGG_MULTI_CONFIG="disagg.multi=1 runs.predictable_replay=1 "
+DISAGG_MULTI_CONFIG="disagg.multi=1 disagg.multi_checksum=1 runs.predictable_replay=1 "
```

Comment:
I know this is unrelated, but could we add 
`runs.rows=100000:500000 and runs.ops=5000000:20000000`

Reply:
Unfortunately `runs.timer` does not work with predictable replay in this setup, so we need to explicitly define `runs.ops`.  `runs.rows` we don't have to keep, that was because of PALM, but we are using PALIte.

Reply:
Got it -- thanks. Changed in `c4fc8725a6`